src/sass/file-preview-card/stories/file-preview-card.stories.js

```diff
@@ -0,0 +1,264 @@
+                <img src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg" alt="Image">
+            </div>
+            <div class="file-preview-card__menu-action">
+                <button class="icon-btn" type="button" aria-label="Open menu">
+                    <svg class="icon icon--16" height="16" width="16" aria-hidden="true">
+                        <use href="#icon-overflow-vertical-16"></use>
+                    </svg>
+                </button>
+            </div>
+            <div class="file-preview-card__label">
+                <button class="icon-btn" type="button" aria-label="Preview">
+                    <svg class="icon icon--12" height="12" width="12" aria-hidden="true">
+                        <use href="#icon-play-16"></use>
+                    </svg>
+                    <span class="file-preview-card__label-text">0:21</span>
+                </button>
+            </div>
+        </div>
+    </div>
+`;
+
+export const generalDoc = () => `
+    <div style="height: 168px; width: 168px;">
+        <div class="file-preview-card">
+            <div class="file-preview-card__center">
+                <svg class="icon" height="40%" width="40%" aria-hidden="true">
+                    <use href="#icon-file-24"></use>
+                </svg>
+            </div>
+            <div class="file-preview-card__menu-action">
+                <button class="icon-btn" type="button" aria-label="Open menu">
+                    <svg class="icon icon--16" height="16" width="16" aria-hidden="true">
+                        <use href="#icon-overflow-vertical-16"></use>
+                    </svg>
+                </button>
+            </div>
+            <div class="file-preview-card__label">
+                <button class="icon-btn" type="button" aria-label="Preview">
+                    <span class="file-preview-card__label-text">CSV</span>
+                </button>
+            </div>
+        </div>
+    </div>
+`;
+
+export const portraitDoc = () => `
+    <div style="height: 168px; width: 168px;">
+        <div class="file-preview-card">
+            <div class="file-preview-card__center">
+                <img src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-portrait-doc.jpeg" alt="Image">
+            </div>
+            <div class="file-preview-card__menu-action">
+                <button class="icon-btn" type="button" aria-label="Open menu">
+                    <svg class="icon icon--16" height="16" width="16" aria-hidden="true">
+                        <use href="#icon-overflow-vertical-16"></use>
+                    </svg>
+                </button>
+            </div>
+            <div class="file-preview-card__label">
+                <button class="icon-btn" type="button" aria-label="Preview">
+                    <span class="file-preview-card__label-text">PDF</span>
+                </button>
+            </div>
+        </div>
+    </div>
+`;
+
+export const landscapeDoc = () => `
```

Comment:
These PDF examples include an image. Let's make them consistent with how it's coded in Skin - for now these should be "other" with the generic icon.

Reply:
I'll remove these PDF examples since the general one should cover it